### PR TITLE
Hugo: Logo section styling refinement

### PR DESCRIPTION
**Pull request by JorindeUsMedia**
- Add intro text for each logo section on homepage
- Improve paddings on logos section

For: https://linear.app/usmedia/issue/CUE-163

### CODE REVIEW

**Comment by JorindeUsMedia** [comment]
I've looked at the changes with Peter and made some styling tweaks.

File: hugo/content/en/_index.html [hunk 1]
```diff
@@ -36,6 +37,7 @@ <h1 class="section__title">alpha.cuelang.org is currently in active development<
 
 {{< logos
     title="Supporters of CUE"
+    intro="Organizations actively support CUE"
```
Comment (Lyoness) [inline hugo/content/en/_index.html:40]: copyedit nit: s/support/supporting

File: hugo/content/en/_index.html [hunk 2]
```diff
@@ -13,6 +13,7 @@ <h1 class="section__title">alpha.cuelang.org is currently in active development<
 
 {{< logos
     title="Users of CUE"
+    intro="Organisations in every industry use CUE to power their software and services"
```
Comment (Lyoness) [inline hugo/content/en/_index.html:16]: @JorindeUsMedia spelling nit: should match the one seen below: Organizations.

File: hugo/content/en/_index.html (hunk 1 above)
Comment (JorindeUsMedia) [reply hugo/content/en/_index.html]: Changed it :)

File: hugo/content/en/_index.html (hunk 2 above)
Comment (JorindeUsMedia) [reply hugo/content/en/_index.html]: Changed it :)

**Review by Lyoness — APPROVED** [review]

**Comment by Lyoness** [comment]
@JorindeUsMedia importing to Gerrit

**Comment by Lyoness** [comment]
https://review.gerrithub.io/c/cue-lang/cuelang.org/+/550791

**Comment by Lyoness** [comment]
Closes [96bd36f](https://github.com/cue-lang/cuelang.org/commit/96bd36f4f7399c21f9cc44cdf7f449a19517d7d2) as merged

**Pull request closed without merging**